package.json

```diff
@@ -48,18 +48,26 @@
       "last 1 safari version"
     ]
   },
+  "browser": {
+    "fs": false,
+    "path": false,
+    "child_process": false
+  },
   "dependencies": {
     "@docsearch/react": "^3.0.0",
     "@headlessui/react": "^1.5.0",
     "@zentered/next-product-docs": "^0.0.127",
```

Comment:
why do we still depend on `@zentered/next-product-docs` here?

Reply:
Ah, I forgot to remove that line.